pkg/restore/restore.go

```diff
@@ -814,6 +823,35 @@ func hasControllerOwner(refs []metav1.OwnerReference) bool {
 	return false
 }
 
+// isComplete returns whether or not an object is considered completed
+// Used to identify whether or not an object should be restored. Only Jobs or Pods are considered
+func isCompleted(obj *unstructured.Unstructured, groupResource schema.GroupResource) (bool, error) {
+	podsGroupResource := schema.GroupResource{Group: "", Resource: "pods"}
+	jobsGroupResource := schema.GroupResource{Group: "batch", Resource: "jobs"}
+	switch groupResource {
+
+	case podsGroupResource:
+		phase, _, err := unstructured.NestedString(obj.UnstructuredContent(), "status", "phase")
+		if err != nil {
+			return false, errors.WithStack(err)
+		}
+		if phase == string(v1.PodFailed) || phase == string(v1.PodSucceeded) {
+			return true, nil
+		}
+
+	case jobsGroupResource:
+		_, found, err := unstructured.NestedString(obj.UnstructuredContent(), "status", "completionTime")
+		if err != nil {
+			return false, errors.WithStack(err)
+		}
+		if found {
```

Comment:
is it possible a job would have an empty `completionTime`? In which case the key might be found, but the job isn't actually complete (i'd assume)? Maybe just check for non-empty value here.